themes/bootstrap3/templates/holds/edit.phtml

```diff
@@ -0,0 +1,97 @@
+<?php
+  // Set page title.
+  $this->headTitle($this->translate('hold_edit_title'));
+
+  // Set up breadcrumbs:
+  $this->layout()->breadcrumbs = '<li><a href="' . $this->url('myresearch-home') . '">' . $this->transEsc('Your Account') . '</a></li>'
+    . '<li><a href="' . $this->url('holds-list') . '">' . $this->transEsc('My Holds') . '</a></li>'
+    . '<li class="active">' . $this->transEsc('hold_edit_title') . '</li>';
+?>
+<h2><?=$this->transEsc('hold_edit_title')?></h2>
+<?php if ($this->helpText): ?>
```

Comment:
Where does this come from? Is it intentionally not escaped or translated?

Reply:
Oops, that was a copy-paste from the new hold form that I used as the base.